src/ContentTypeMiddleware.php

```diff
@@ -18,6 +17,8 @@ final class ContentTypeMiddleware implements MiddlewareInterface
     private MiddlewareInterface $negotiator;
     private StreamFactoryInterface $streamFactory;
 
+    public const NOT_ACCEPTABLE = 'NOT ACCEPTABLE';
```

Comment:
I'm struggling a bit with this... I'll change things to use the FQCN of the formatter to avoid having this defined.